src/app/settings/views/Configuration/GeneralForm/GeneralForm.tsx

```diff
@@ -111,6 +116,90 @@ const GeneralForm = (): JSX.Element => {
         type="text"
         wrapperClassName="u-sv2"
       />
+      <p>MAAS theme main colour</p>
+      <Row className="themeRow">
+        <Col medium={1} size={1} small={2}>
+          <FormikField
+            colour="default"
+            component={ThemedRadioButton}
+            label="Default"
+            name="theme"
+          />
+        </Col>
+        <Col medium={1} size={1} small={2}>
+          <FormikField
+            colour="bark"
+            component={ThemedRadioButton}
+            label="Bark"
+            name="theme"
+          />
+        </Col>
```

Comment:
This section seems like a good use case for mapping through a list to reduce repetition.

```
["bark", (...) ].map(color => (<Col medium={1} size={1} small={2}>
          <FormikField
            color={color}
            component={ThemedRadioButton}
            label="Bark"
            name="theme"
          />
```

Might  be worth using the American spelling `color` throughout to match CSS `color` - it'll be consistent with most of the codebase.